web3/main.py

```diff
@@ -524,6 +527,8 @@ def persistent_websocket(
 class _PersistentConnectionWeb3(AsyncWeb3):
     provider: PersistentConnectionProvider
 
+    # w3 = AsyncWeb3.persistent_websocket(provider)
```

Comment:
I like these examples of usage. Would a label, something like `# Example usage:` or `# Access this method with pattern:` make it more clear for users?

Reply:
I'd like to leave that up to the documentation. This is more so we separate the logic for each of those design patterns into code blocks.